lib/applicationMocks.ts

```diff
@@ -3,7 +3,7 @@ import { cookies } from 'next/headers'
 const loginCookie = 'connect_session_example_user'
 
 export const setUserEmail = (email: string) =>
-  cookies().set(loginCookie, email, { maxAge: 3600 * 24, sameSite: 'strict' })
```

Comment:
with `strict` cookies, we can't redirect back from another website. We need `lax` (which is the default) to achieve that, and that's what we have in Connect as well